exercises/concept/boutique-inventory/boutique_inventory_test.rb

```diff
@@ -6,19 +6,26 @@ def test_no_item_names
     assert_empty BoutiqueInventory.new([]).item_names
   end
 
-  def test_one_item_name
+  def test_one_item_name_out_of_stock
```

Comment:
This changes the test.  And is related to the above as well.  Getting an item name should not depend on stock level, as makes sense intuitively since this test name has changed.  It may be just coincidentally that the item name happens to be out of stock.  It should be as relevant to test getting one item name given any level of stock.